README.md

```diff
@@ -163,57 +180,28 @@ and select the profiles you want to build.
-
-After this command runs, images can be found in your preferred container engine image list.
 
-### Eclipse IDE
-The simplest way to start developing on Eclipse Kura™ is to use an [Eclipse Installer](https://www.eclipse.org/downloads/) based setup. A detailed installation and setup guide is available on the [official documentation](https://eclipse.github.io/kura/docs-develop/java-application-development/development-environment-setup). Here you'll find a brief explaination of the required steps.
+> [!NOTE]
+This build step requires 'docker' to be a executable command on your system. For Instance, if you are using Podman please follow the [Emulating Docker Cli Guide](https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman) before running the command above.
 
-To correctly setup the environment, proceed as follows:
-- Install a jdk-8 distribution like [Eclipse Temurin](https://adoptium.net/temurin/releases/?version=8) for your specific CPU architecture and OS.
-- Start the Eclipse Installer
-- Switch to advanced mode (top right hamburger menu > Advanced Mode)
-- Select "Eclipse IDE for Eclipse Committers" and configure the "Product Version" to be the version **2023-03 or newer**.
-- Select the Eclipse Kura™ installer from the list. If this is not available, add a new installer from https://raw.githubusercontent.com/eclipse/kura/develop/kura/setups/kura.setup, then check and press the Next button
-- Select the "Developer Type":
-  - "User": if you want to develop applications or bundles running on Kura™, select this option. It will install only the APIs and the examples.
-  - "Developer" : if you are a framework developer, select this option. It will download and configure the Eclipse Kura™ framework.
-- Update Eclipse Kura™ Git repository username and customize further settings if you like (e.g. Root install folder, Installation folder name). To show these options, make sure that the "Show all variables" checkbox is enabled
-- Set the `JRE 1.8 location` value to the installed local jdk-8 VM
-- Leave all Bootstrap Tasks selected and press the Finish button
-- Accept all the licenses and wait for the installation to finish
-At first startup Eclipse IDE will checkout the code, perform a full build and configure a few Working Sets
-- When the tasks are completed. In the IDE open (double click) Target Platform > Target-Definition > Kura™ Target Platform Equinox 3.16.0, and press "Set as Target Platform" located at the top right of the window
-
-Now you are ready to develop on Eclipse Kura™.
-
-To raise an issue, please report a bug on [GitHub issues](https://github.com/eclipse/kura/issues/new).
+After this command runs, images can be found in your preferred container engine image list.
```

Comment:
After the command run?

Reply:
Addressed in: 53fc4ac471ea81e481b9d015959ecec57240273b